mailpoet/lib/Automation/Engine/Registry.php

```diff
@@ -16,6 +17,9 @@ class Registry {
   /** @var array<string, Subject<Payload>> */
   private $subjects = [];
 
+  /** @var SubjectTransformer[] */
+  private $subjectTransformer = [];
```

Comment:
Shouldn't this be called `$subjectTransformers` (and the getter `getSubjectTransformers`) in accordance with other naming?

Reply:
c4cde16a9